### ISSUE TITLE
WikidPad V2.2 does not run under Window 11

### ISSUE THREAD

**Issue by Spitzenamarone** [WikidPad/WikidPad#89]
I tried to run V2.2 under Win11, for a moment ist is listed in taskmanager and then i disappears, no error message happen.

V2.4 alpha run but an error messag happen on closing

**Comment by benkinooby** [comment]
@Spitzenamarone I don't think that any ressources will be spent on fixing V2.2, if newer versions (V2.3 and V2.4 alpha) are available. Can you post the error message of V2.4 alpha?
Would using V2.3 be an option for you?

**Comment by benkinooby** [comment]
Closing this issue lacking response. Happy to reopen if so desired.
